app/views/simulations/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @simulation.simulation_standings.size == 0 %>
+    <p>Simulating still. Come back later!</p>
```

Comment:
If you're interested in i18n, this may be an opportunity to use Rails' i18n to get exposure to how that works. 

Maybe rewrite this so there's an `en` translation and a `pt-BR` translation, and get a feel for how that works.

Reply:
Yes. Created #11 to address this.